src/main/java/core/basesyntax/SkillOfDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface SkillOfDrawing {
+    void ableToDraw();

```

Comment:
Please see [this](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names) article about method names